src/fetchTodo.js

```diff
@@ -0,0 +1,23 @@
+const AWS = require("aws-sdk");
+const dynamo = new AWS.DynamoDB.DocumentClient();
+
+const fetchTodo = async (event) => {
+  const { id } = event.pathParameters;
+  let todo;
```

Comment:
Why do you declare this variable here? The best practice is to always use const and only if there is no other way to use let